engine/src/lib.rs

```diff
@@ -946,15 +1161,18 @@ mod contract {
             // that they over paid for their deposit.
             unsafe { io.promise_create_batch(&promise) };
         }
+        update_hashchain(&mut io, function_name!(), &input, &[], &Bloom::default());
```

Comment:
This function also has a [non-empty output](https://github.com/aurora-is-near/aurora-engine/blob/develop/engine/src/connector.rs#L539). And similarly for the other `storage_*` functions.

Reply:
As per our conversation, capturing the output here and in others would require an investigation of how they look on the standalone side (get_output_and_log_bloom method).